### DISCUSSION TITLE
Resizeable Modern Suggestion

### DISCUSSION THREAD

**Discussion by Haftago** [runelite/runelite#16799] [Ideas]
https://i.imgur.com/A8fFzjx.png 
https://i.imgur.com/y0HzMTa.png

Currently when on resizeable modern, theres an empty spot and logout gets sent to the top of screen attached to minimap. 
Please make it so logout can be back into tabs like all other layouts.

**Comment by Nightfirecat** [comment]
That is how the interfaces are constructed by Jagex, we can't control that.
